components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,54 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File to TMP",
+  description: "Download a specific file to the temporary directory. [See the documentation](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor).",
+  key: "dropbox-download-file-to-tmp",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    dropbox,
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFile",
+      ],
+      description: "The file path in the user's Dropbox to download.",
+    },
+    name: {
+      type: "string",
+      label: "File name",
+      description: "The new name of the file to be saved, including it's extension. e.g: `myFile.csv`",
```

Comment:
```suggestion
      description: "Your new file name",
```

Reply:
Removed period at the end of the sentence to follow Google's Developer docs style guide (https://developers.google.com/style/periods)